src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+const coordsWall = wall.getBoundingClientRect();
+const coordsSpider = spider.getBoundingClientRect();
+
+spider.style.left = `${coordsWall.width / 2 - coordsSpider.width / 2}px`;
+spider.style.top = `${coordsWall.height / 2 - coordsSpider.width / 2}px`;
```

Comment:
There is a mistake in the calculation of the vertical position of the spider. The line `spider.style.top = `${coordsWall.height / 2 - coordsSpider.width / 2}px`;` should use `coordsSpider.height` instead of `coordsSpider.width` to correctly center the spider vertically within the wall.